src/main/java/frc/robot/Robot.java

```diff
@@ -34,7 +35,11 @@ public class Robot extends TimedRobot {
     MotorController driveLeftChild = RobotConfigs.getLeftChild();
     MotorController driveRightParent = RobotConfigs.getRightParent();
     MotorController driveRightChild = RobotConfigs.getRightChild();
+    public static MotorController leftShooterMotor = MotorControllerFactory.create(6, MotorController.Type.SparkMax);
```

Comment:
The public static part was a temporary solution, so that we could physically test the shooter.